editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -65,7 +65,7 @@ describe('React Render Count Tests -', () => {
 
     const renderCountAfter = renderResult.getNumberOfRenders()
     // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`457`)
+    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`770`)
```

Comment:
one thing I'd call out here: if a component change / user facing feature changes these numbers, that's a potential regression. here we are basically moving the baseline. we are very certain that the actual performance did not change, just the way we count this number

Reply:
I'd argue that any change to these numbers is a potential regression. So especially if we're changing something that _isn't_ a UI change we need to be sure that the updated figures are not a regression

Reply:
It's also the baseline _with the test dispatch_, which in this case is a key part of the difference.